salesforce-analytics-cloud/3.5/modules/ROOT/pages/salesforce-analytics-connector-reference.adoc

```diff
@@ -0,0 +1,587 @@
+| Object Store a| String |  A reference to the object store to use to store each resource owner ID's data. If not specified, the Mule automatically provisions the default object store. |  |
+|===
+
+[[salesforce-analytics_oauth-jwt]]
+=== OAuth JWT
+
+
+==== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| TLS configuration a| <<Tls>> |  If the protocol is configured as HTTPS, then you must also configure the keystore configuration. |  |
+| Read Timeout a| Number |  Specifies the amount of time, in milliseconds, that the consumer waits for a response before it times out. The default value is `0`, which means infinite. |  0 | 
+| Read Timeout Time Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS | Time unit that qualifies the Read Timeout. |  MILLISECONDS | 
+| Connection Timeout a| Number |  Specifies the amount of time, in milliseconds, that the consumer attempts to establish a connection before it times out. The default value is `0`, which means infinite. |  0 | 
+| Connection Timeout Time Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time Unit that qualifies the Connection Timeout. |  MILLISECONDS | 
+| Host a| String |  Host name of the proxy. Required if a proxy is used. If this property is not set, then no proxy is used.  |  |
+| Port a| Number |  Port of the proxy. If a Host is set, this property must also be set, and cannot be a negative number. |  |
+| Username a| String |  Username used to authenticate against the proxy. Required if using authentication against the proxy. If this property is not set, then no authentication is used against the proxy. |  |
+| Password a| String |  Password used to authenticate against the proxy. |  |
+| Consumer Key a| String |  Consumer key for the Salesforce-connected app. |  | *x*
+| Key Store a| String |  Path to the keystore used to sign data during authentication. |  | *x*
+| Store Password a| String | Password of the keystore. |  | *x*
+| Principal a| String | Username of the Salesforce user on whose behalf to take action.  |  | *x*
+| Token Endpoint a| String a| URL pointing to the server responsible for providing the authentication token. According to Salesforce, this URL is `+https://login.salesforce.com/services/oauth2/token+`. If implementing for a community, the URL is `+https://acme.force.com/customers/services/oauth2/token+`, where `acme.force.com/customers` is your own community URL. |  `+https://login.salesforce.com/services/oauth2/token+` | 
+| Audience Url a| String a| The audience identifies the authorization server as an intended audience. The authorization server must verify that it is an intended audience for the token. 
```

Comment:
Not sure what this means: "The audience identifies the authorization server as an intended audience." I think you should start with "Identifies the authorization server as an intended audience."